### DISCUSSION TITLE
Postgres version update

### DISCUSSION THREAD

**Discussion by Dulanic** [tteck/Proxmox#3523] [Q&A]
Is there any automated upgrading that is done? It seems like no, because I find it shows 15 and 16 installed though it connects to 15. I want to upgrade it to 16, but I also don't want to mess up the container, so I just wanted to ensure here before I did this, that there is no automated method of upgrading and that I am good upgrading in place.

![image](https://github.com/user-attachments/assets/9f60762e-5913-4f99-9142-021ca8dea4f5)

**Comment by tteck** [comment]
The script was modified almost a year ago to install v16 https://github.com/tteck/Proxmox/commit/6df6d1c2fb27c5480e64e7aba8908b4cae12eafe

**Reply by Dulanic** [reply]
OK thanks, but if it was originally installed at 15, it will stay 15, correct? I assume that you did not include any pg_upgrade stuff etc to upgrade in place? Doesn't seem so, which I also wouldn't expect, was just making sure. I will upgrade the container manually if so.
